examples/step-69/step-69.cc

```diff
@@ -1597,23 +1600,43 @@ namespace Step69
   }
 
   // We conclude this section by defining static arrays
-  // <code>component_names</code> that contain strings describing the
-  // component names of our state vector. We have template specializations
-  // for dimensions one, two and three, that are used later in DataOut for
-  // naming the corresponding components:
+  // <code>component_names</code> and <code>component_physical_units</code>
+  // that contain strings describing the
+  // components of our state vector, as well as the physical units of
+  // these quantities that can be used to annotate the VTU output
+  // files we generate for visualization and postprocessing. We have
+  // template specializations for dimensions one, two and three, that
+  // are used later in DataOut for naming the corresponding
+  // components:
 
   template <>
   const std::array<std::string, 3> ProblemDescription<1>::component_names{
     {"rho", "m", "E"}};
 
+  template <>
+  const std::array<std::string, 3>
+    ProblemDescription<1>::component_physical_units{
+      {"kg/m", "kg*m/s/m", "J/m"}};
+
+
   template <>
   const std::array<std::string, 4> ProblemDescription<2>::component_names{
     {"rho", "m_1", "m_2", "E"}};
 
+  template <>
+  const std::array<std::string, 4>
+    ProblemDescription<2>::component_physical_units{
+      {"kg/m/m", "kg*m/s/m/m", "kg*m/s/m/m", "J/m/m"}};
+
   template <>
   const std::array<std::string, 5> ProblemDescription<3>::component_names{
     {"rho", "m_1", "m_2", "m_3", "E"}};
 
+  template <>
+  const std::array<std::string, 5>
+    ProblemDescription<3>::component_physical_units{
+      {"kg/m/m/m", "kg*m/s/m/m/m", "kg*m/s/m/m/m", "kg*m/s/m/m/m", "J/m/m/m"}};
```

Comment:
Is there a good reason to use this format instead of
```suggestion
      {"kg/m^3", "kg/s/m^2", "kg/s/m^2", "kg/s/m^2", "J/m^3"}};
```
?

Reply:
The problem is that the unyt framework wants to use `**` for exponentiation, rather than `^`, so I opted for the unambiguous approach.

Reply:
OK.